pkg/provider/kubernetes/crd/kubernetes_http.go

```diff
@@ -85,9 +142,11 @@ func (p *Provider) loadIngressRouteConfiguration(ctx context.Context, client Cli
 			}
 
 			normalized := provider.Normalize(makeID(ingressRoute.Namespace, serviceKey))
-			serviceName := normalized
+			serviceName := ""
```

Comment:
Here I would have kept the default value to normalized and add an else case for intermediate IngressRoutes which set the servicename to empty with a comment.

Reply:
Good catch, it's definitely clearer!